tests/webview/ui/test_content.py

```diff
@@ -320,7 +320,7 @@ def test_share_on_top_right_corner(webview_base_url, selenium):
             None,
             None,
         ),
-        ("e42bd376-624b-4c0f-972f-e0c57998e765", "mitosis genetics", True, 0, True, False),
```

Comment:
I think we already have one above that doesn't return results. this test should be set to `True` and have a term that gets results.